applications/incident-management/app/src/main/java/lh/quickstart/App.java

```diff
@@ -0,0 +1,105 @@
+package lh.quickstart;
+
+import java.io.IOException;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import io.grpc.Status.Code;
+import io.littlehorse.sdk.common.config.LHConfig;
+import io.littlehorse.sdk.worker.LHTaskWorker;
+import io.littlehorse.sdk.common.proto.LHPublicApiGrpc;
+import io.littlehorse.sdk.common.proto.LHPublicApiGrpc.LHPublicApiBlockingStub;
+import io.littlehorse.sdk.common.proto.PutExternalEventDefRequest;
+
+public class App {
+
+    // Task identifiers
+    private static final String VERIFY_TASK = "verify-incident";
+    private static final String PERIODIC_CHECK_TASK = "periodic-check-task";
+    private static final String EXTERNAL_EVENT_NAME = "incident-resolved";
+
+    // Method to register the external event with LittleHorse
+    private static void registerExternalEventDef(LHPublicApiBlockingStub client) {
+        System.out.println("Registering external event " + EXTERNAL_EVENT_NAME);
+        try {
+            // Attempt to create a new external event definition
+            client.putExternalEventDef(
```

Comment:
Good work! We need to make a client wrapper which does some of this for you. If littlehorse did our job, it would be one line of code :see_no_evil: